java/code/src/com/suse/manager/reactor/messaging/JobReturnEventMessageAction.java

```diff
@@ -248,6 +248,11 @@ public void execute(EventMessage msg) {
                 jobReturnEvent.getMinionId());
         if (minion.isPresent()) {
             MinionServer m = minion.get();
+            if (jobResult.isEmpty()) {
+                LOG.error("Do not update server info since job=" +  jobReturnEvent.getJobId() + " in minion=" +
```

Comment:
I am not sure if this should actually log as `error` here? Maybe for the context of not updating the server info, a log level of `info` could be enough, assuming that the actual failure of the job had been logged as `error` elsewhere.

Reply:
`info` is not logged by default. But maybe `warn` ?

Reply:
changed in warn